docs/typescript.mdx

```diff
@@ -2,10 +2,23 @@
 title: 'TypeScript'
 ---
 
-Emotion includes TypeScript definitions for `@emotion/react` and `@emotion/styled`. These definitions also infer types for css properties with the object syntax, HTML/SVG tag names, and prop types.
+Emotion includes TypeScript definitions for `@emotion/react` and `@emotion/styled`. These definitions infer types for css properties with the object syntax, HTML/SVG tag names, and prop types.
 
 ## @emotion/react
 
+The easiest way to use the css prop with TypeScript is with the new JSX
+transform and the `jsxImportSource` TSConfig option. For this approach, your
+TSConfig `compilerOptions` should contain
+
+```json
+"jsx": "react-jsx",
+"jsxImportSource": "@emotion/react"
+```
+
+For most users, this is all the setup that is required. You can now define
+styles using the object syntax or template literal syntax and pass them to your
+components via the `css` prop.
```

Comment:
I wonder how this line-wrapping affects the rendering of our docs site. It feels like it would be more straightforward if we wouldn't wrap lines in `.mdx?` files and have that job done by the browser, automatically.

Reply:
Hard-wrapping the lines in the .mdx does not affect what renders in the browser assuming there is not a double space at the end of any line. But I will change to have it all on one line.